webapp/src/modules/authorization/actions.js

```diff
@@ -0,0 +1,145 @@
+import { buildTransactionAction } from 'modules/transaction/utils'
+
+// Fetch authorization
+
+export const FETCH_AUTHORIZATION_REQUEST = '[Request] Fetch Authorization'
+export const FETCH_AUTHORIZATION_SUCCESS = '[Success] Fetch Authorization'
+export const FETCH_AUTHORIZATION_FAILURE = '[Failure] Fetch Authorization'
```

Comment:
Do you think this should be pluralized?

We're fetching multiple authorizations for a user, but the module is called authorization and contains multiple instances of the same authorization instance for each address.

I'm not sure if using `authorization.allowances, authorization.approvals` is better than using `authorizations.allowances, authorizations.approvals` everywhere

Reply:
We have `transaction` for transactions and `parcels` for parcel. As we should need to be consistent when naming, I prefer plural for `authorizations` instead of `authorization`. 

The actions are ok if you are always fetching only one authorization

Reply:
The thing is, the module can be singular because is the _name_ of the module (I think we're using those and being consistent on the other dapps) @cazala back me up if I'm not correct

The problem I see here with `FETCH_AUTHORIZATION_REQUEST` is that one authorization consists of several things, multiple allowances and approvals, so it might be a confusing action name

Reply:
back in the day we decided to use singular names for modules because it's more aligned with DDD: it's the name of the module, ie "the transaction module", "the parcel domain". But we  screwed up and let a few plurals slip in on this dapp /shrug.

Reply:
lets leave it like this 👍